### ISSUE TITLE
Multiple Turbo-pack repositories

### ISSUE THREAD

**Issue by pyscripter** [TurboPack-Tokyo/SynEdit#1]
Thanks a lot for the great work done in maintaining such useful Delphi components.

Here  is a suggestion.

I find it (and others may too) confusing the fact that there a are multiple TurboPack repositories e.g.  TurboPack, TurboPack-Seattle, TurboPack-Tokyo etc.  It also makes it difficult to backport fixes etc. and reduces the visibility of each repository.   Also I found that the latest Turbopack is typically compatible with most recent versions of Delphi all the way back to XE7 and maybe more.

I think a better idea would be have just one repository, say TurboPack and every time a new Delphi version comes out to have a new release.  So users of earlier version of Delphi will have access to components that are guaranteed to be compatible with their Delphi version, but at the same time they can use the master branch if it is still compatible with their version.

**Comment by romankassebaum** [comment]
Yes, that would be better. I started to support the latest "city versions" (Tokyo, Berlin and Seattle) some months ago but I'm currently too busy with commercial projects.
